### DISCUSSION TITLE
아바타 옆에 표시되는 챗/이름 글씨 크기에 최소 크기가 있으면 좋을거 같아요.

### DISCUSSION THREAD

**Discussion by jihwanKo** [metamong-friends/metamong-friends.github.io#17] [제안하기]
쿼터뷰에서 회의실 안에 사람들을 다 보려면 어느정도 줌 아웃을 해야 하는 경우가 있는데, 
이때 아바타 옆으로 표시되는 "이름"과 "챗"의 폰트 크기가 좀 더 컸으면 좋겠습니다. 
줌 아웃을 많이 해도 폰트 크기가 일정 이상 줄어들지 않으면 좋을거 같아요.
특히 숄더뷰로 보면 멀리 있는 사람의 글씨가 거의 안보이는 경우가 있어서요.

**Comment by shlee303** [comment]
아래 글씨 폰트가 너무 작다는 의견이 있었는데, 함께 조치 해보겠습니다, 의견 감사합니다~!

**Comment by shlee303** [comment]
@jihwanKo 줌아웃 하여도 일정 크기 유지하도록 조치 완료하였습니다! 챗 등 기본 폰트 크기도 확대했습니다. 좋은 의견 감사합니다!
